code/iaas/lb-logic/src/main/java/io/cattle/iaas/lb/api/action/LoadBalancerAddHostActionHandler.java

```diff
@@ -38,17 +31,8 @@ public Object perform(String name, Object obj, ApiRequest request) {
         LoadBalancer lb = (LoadBalancer) obj;
         long hostId = DataAccessor.fromMap(request.getRequestObject()).withKey(LoadBalancerConstants.FIELD_LB_HOST_ID)
                 .as(Long.class);
-        createLbHostMap(lb.getId(), hostId);
+        lbService.addHostToLoadBalancer(lb, hostId);
 
         return objectManager.reload(lb);
```

Comment:
Since addHostToLoadBalancer just creates the mapping entry, does it really change the lb instance?  I'm wondering whether the reload is necessary or not?